src/domain/shared/layout/EntityPageSection.ts

```diff
@@ -3,6 +3,7 @@ export enum EntityPageSection {
   Settings = 'settings',
   About = 'about',
   Contribute = 'contribute',
+  Explore = 'explore',
```

Comment:
@me-andre Do you think this should be added somewhere else maybe? Maintained as a separate enum?